packages/components/pages/f-checkout/README.md

```diff
@@ -85,6 +85,8 @@ The props that can be defined are as follows:
 | `loginUrl` | `String` | `-` | URL to navigate to if the user wishes to change account. |
 | `paymentPageUrlPrefix` | `String` | `-` | URL prefix to navigate to after the order has been successfully placed, so the user can pay. The `orderId` will be appended to this URL to form the full URL. |
 | `applicationName` | `String` | `-` | The name of the application using this component. |
+| `getNoteConfigUrl` | `String` | - | URL for the API called to get the note configuration for split notes |
+| `checkoutFeatures` | `Object` | - | Object containing relevant feature flags set in CoreWeb configuration files |
```

Comment:
Is it likely we'll need to add any browser / visual tests for these props? For example if you supply the `checkoutFeatures` object, can that change the UI state at all?